### PR TITLE
Modification of 5_CUR.ipynb

### PR DESCRIPTION
**Pull request by PKGuo**
Fixed some typos and added some details to help readers understand.

### CODE REVIEW

**Comment by rosecers** [comment]
> Refined 5_CUR.ipynb for clarity.

Your PR summaries should be more explicit. State what types of exact changes you made and why.

File: notebooks/5_CUR.ipynb [hunk 1]
```diff
@@ -162,10 +162,12 @@
     "For the $j^{th}$ column, the importance score is given by\n",
     "\n",
     "\\begin{equation}\n",
-    "\\pi_j = \\sum_i^k (U_C)_{ij}^2,\n",
+    "\\pi_j = \\sum_j^k (U_C)_{ij}^2,\n",
```
Comment (rosecers) [inline notebooks/5_CUR.ipynb]: This is not correct. It is the sum over `i` corresponding to column `j`. If we sum over `j` we're finding the importance of `i`.

File: notebooks/5_CUR.ipynb [hunk 2]
```diff
@@ -162,10 +162,12 @@
...
     "\\end{equation}\n",
     "\n",
-    "where $k$ is again the target rank of the new matrix and $(U_C)_{ij}$ is the value in the $i^{th}$ row and $j^{th}$ column of $\\mathbf{U}_C$."
+    "where $k$ is again the target rank of the new matrix and $(U_C)_{ij}$ is the value in the $i^{th}$ row and $j^{th}$ column of $\\mathbf{U}_C$.\n",
+    "\n",
+    "Note that `sps.linalg.svds` actually returns $U_C^T$ having right singular vectors as rows. So we use `sum (axis = 0)` below."
```
Comment (rosecers) [inline notebooks/5_CUR.ipynb]: This will not be necessary when you incorporate the new code.

**Review by rosecers — CHANGES_REQUESTED** [review]

File: notebooks/5_CUR.ipynb (hunk 2 above)
Comment (PKGuo) [reply notebooks/5_CUR.ipynb]: Okay, I will remove it.

File: notebooks/5_CUR.ipynb (hunk 1 above)
Comment (PKGuo) [reply notebooks/5_CUR.ipynb]: I will fix it.

File: notebooks/5_CUR.ipynb [hunk 3]
```diff
@@ -162,12 +162,10 @@
     "For the $j^{th}$ column, the importance score is given by\n",
     "\n",
     "\\begin{equation}\n",
-    "\\pi_j = \\sum_j^k (U_C)_{ij}^2,\n",
+    "\\pi_j = \\sum_i^k (U_C)_{ji}^2,\n",
```
Comment (rosecers) [inline notebooks/5_CUR.ipynb:165]: Please rebase this so that it doesn't require a new commit.

**Pull request closed without merging**